tools/build/mk/OptionalObsoleteFiles.inc

```diff
@@ -1659,16 +1399,167 @@ OLD_FILES+=usr/share/dict/web2a
 OLD_FILES+=usr/share/dict/words
 OLD_DIRS+=usr/share/dict
 .endif
-
-.if ${MK_DMAGENT} == no
-OLD_FILES+=etc/dma/dma.conf
-OLD_DIRS+=etc/dma
-OLD_FILES+=usr/libexec/dma
-OLD_FILES+=usr/libexec/dma-mbox-create
-OLD_FILES+=usr/share/man/man8/dma.8.gz
-OLD_FILES+=usr/share/examples/dma/auth.conf
-OLD_FILES+=usr/share/examples/dma/mailer.conf
-OLD_DIRS+=usr/share/examples/dma
+
+.if ${MK_DMAGENT} == no
```

Comment:
what happened here?

Reply:
The old file doesn't have superfluous whitespace, and the new one doesn't seem to have either...